composer.json

```diff
@@ -81,7 +82,6 @@
         "aws/aws-sdk-php": "^3.155",
         "doctrine/dbal": "^2.6|^3.0",
         "filp/whoops": "^2.8",
-        "guzzlehttp/guzzle": "^6.5.5|^7.0.1",
```

Comment:
Please revert these changes to the dependencies.

Reply:
Yup sorry my bad, just pushed the revert as you were commenting.

Reply:
I'll have a look at the failing tests too.